src/structures/idt.rs

```diff
@@ -663,7 +705,8 @@ macro_rules! impl_set_handler_fn {
             /// further customization.
```

Comment:
We should document here that the method is only available with the `abi_x86_interrupt` feature and point to the `set_handler_addr` function as an alternative. For example:

```suggestion
            /// further customization.
            ///
            /// This method is only usable with the `abi_x86_interrupt` feature enabled. Without it, the
            /// unsafe [`Entry::set_handler_addr`] method has to be used instead.
```